packages/aws-cdk/lib/init-templates/app/python/README.md

```diff
@@ -0,0 +1,42 @@
+
+Welcome to your CDK Python project!
+
+You should explore the contents of this template. It demonstrates a CDK app with two instances of
+a stack (`HelloStack`) which also uses a user-defined construct (`HelloConstruct`).
+
+The `cdk.json` file tells the CDK Toolkit how to execute your app. It uses a script called `app.sh`
+to do that.
+
+This project is set up like a standard Python project.  The initialization process also creates
+a virtualenv within this project, stored under the .env directory.
+
+After the init process completes, you can use the following steps to get your project set up.
+
+```
+$ source .env/bin/activate
+$ pip install -r requirements.txt
+$ python setup.py develop
```

Comment:
Isn't `develop` for libraries (as opposed to apps)?

Reply:
`develop` is just a variant of `install` that allows you to edit your code and run it without having to re-install.  This example really consists of both an app and a library.  I have restructured the code slightly to make that more clear, I hope.